docs/3x/get_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+1. [Quick Samples](#quick-samples)
+
+2. [Feature Matrix](#feature-matrix)
+
+## Quick Samples
+
+```shell
+# Install Intel Neural Compressor
+pip install neural-compressor-pt
+```
+```python
+from transformers import AutoModelForCausalLM
+from neural_compressor.torch.quantization import RTNConfig, prepare, convert
+
+user_model = AutoModelForCausalLM.from_pretrained("EleutherAI/gpt-neo-125m")
+quant_config = RTNConfig()
+prepared_model = prepare(model=user_model, quant_config=quant_config)
+quantized_model = convert(model=prepared_model)
+```
+
+## Feature Matrix
+Intel Neural Compressor 3.X aim to provide framework extension APIs for the popular deep learning frameworks PyTorch and TensorFlow. 
+The below table provides a quick overview of the APIs available in Intel Neural Compressor 3.X.
+The new APIs mainly focus on quantization related features especially for the algorithms benefit LLM inference. 
+It also provides some common APIs across different frameworks. For example, Auto Tune API support accuracy driven quantization and mixed precision, benchmark API aimed to measure the multiple instances performance of the quantized model.
```

Comment:
I feel `module` is more appropriate than `APIs`. BTW, `common` is invisible for users.

Reply:
Agree, I will change APIs into module. And here the common APIs not related to the common folder, just means auto-tune and benchmark which is common used across frameworks. To avoid confuse, I will update into others.